test/unit/au/org/emii/portal/DownloadReportTests.groovy

```diff
@@ -61,37 +61,21 @@ class DownloadReportTests extends GrailsUnitTestCase {
     void testAddFileEntry() {
 
         assertEquals 0, report.numberOfFilesTried
-        assertEquals "", report.reportBody
 
-        report._addFileEntry "url", "filename", "went well"
+        def header = report.reportTempFile.getText()
+
+        report._addFileEntry(report._makeFileEntry("url", "filename", "went well"))
 
         assertEquals 1, report.numberOfFilesTried
-        assertEquals """\
 
+        def entry =  """\
             --[ #1 ]------------------------------------
             URL:                 url
             Filename in archive: filename
             Result:              went well
-            """,
-            report.reportBody
-    }
-
-    void testGetText() {
```

Comment:
how come you removed this test?

Reply:
This was a test for the text getter that no longer exists.  We now get a file instead.

Reply:
I've added a test for getTempFile - tests that the file exists and is readable